npu.user.js

```diff
@@ -1102,6 +1102,12 @@
           }
         });
 
+        /* From most to least important */
```

Comment:
Perhaps you could add a little explanation **why** precedence is needed.

Reply:
Done.